shared/source/os_interface/linux/drm_neo.cpp

```diff
@@ -659,15 +659,17 @@ std::vector<std::unique_ptr<HwDeviceId>> Drm::discoverDevices(ExecutionEnvironme
 
     do {
         const char *renderDeviceSuffix = "-render";
+        constexpr size_t renderDevSufSize = std::char_traits<char>::length(renderDeviceSuffix);
+        constexpr size_t pciDevDirLen = std::char_traits<char>::length(Os::pciDevicesDirectory);
```

Comment:
`Os::pciDevicesDirectory` is not known at compile time as it is defined differently for ULT and for runtime, hence the size cannot be constexpr

Reply:
fixed in https://github.com/intel/compute-runtime/pull/791/commits/56f6655c6dbe859081d8d932513d9010050d64ab